commander3/src/comm_tod_mapmaking_mod.f90

```diff
@@ -373,13 +374,16 @@ subroutine compute_Ax(tod, x_imarr, pmask, comp_S, M_diag, x, y, x_in, y_out)
       real(sp),     dimension(0:),     intent(in)              :: pmask
       logical(lgt), intent(in)                                 :: comp_S
       real(dp),                dimension(:,:), intent(in) :: M_diag
+      integer(i4b), intent(in)         :: split
       real(dp),                dimension(1:,0:), intent(inout), optional :: x
       real(dp),                dimension(1:,0:), intent(inout), optional :: y
       real(dp),     dimension(0:, 1:), intent(in),    optional :: x_in
       real(dp),     dimension(0:, 1:), intent(inout), optional :: y_out
 
       integer(i4b), allocatable, dimension(:)         :: flag
       integer(i4b), allocatable, dimension(:, :)      :: pix, psi
+      integer(i4b), dimension(10) :: t_arr=(/52131,52496,52861,53227,53592,53957,54322, &
+                                          &  54688,55053,55418/)
```

Comment:
Hard coded array?

Reply:
These are MJDs corresponding to each year of WMAP operation. Maybe worth setting as a parameter somewhere.